packages/vetrina/src/Vetrina/Vetrina.purs

```diff
@@ -305,7 +305,7 @@ render (Self props state setState) components = vetrinaContainer state.purchaseS
               components.accountForm u
             -- Can't do much without a user
             Nothing -> props.unexpectedError
-        AuthenticationError -> components.newPurchase
+        AuthenticationError _ -> components.newPurchase
```

Comment:
Now it's the alignment again.

Reply:
Fixed! Sorry, I started aligning them and tried to fixed them all but that didn't quite work :D It's now fixed.